### PR TITLE
test: License v3 - add test cases for empty limitations

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: ee/packages/license/src/license.spec.ts [hunk 1]
```diff
@@ -22,6 +22,19 @@ it('should not prevent if the counter is under the limit', async () => {
 	await expect(licenseManager.shouldPreventAction('activeUsers')).resolves.toBe(false);
 });
 
+it('should not prevent actions there is no limit set in the license', async () => {
```
Comment (sampaiodiego) [inline ee/packages/license/src/license.spec.ts:25]:
```suggestion
it('should not prevent actions if there is no limit set in the license', async () => {
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Review by sampaiodiego — APPROVED** [review]
